SwrveSDK/SDK/Talk/SwrveBaseCampaign.h

```diff
@@ -32,6 +32,7 @@
 @property (nonatomic, retain) NSDate* showMsgsAfterLaunch;          /*!< Timestamp to block messages after launch. */
 @property (atomic)            bool messageCenter;                   /*!< Flag indicating if it is a Message Center campaign. */
 @property (nonatomic, retain) NSString* subject;                    /*!< Message Center subject of the campaign. */
+@property (nonatomic, retain) NSDate* dateStart;                    /*!< Timestamp representing the start date of Message Center campaign. */
```

Comment:
This is exposed for all campaigns types so comment should be:
 /*!< Timestamp representing the start date of campaign. */